examples/multi-gpu-movielens/01-03b-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-TensorFlow.ipynb

```diff
@@ -698,110 +710,57 @@
-      "[1,1]<stderr>:2021-04-06 09:25:23.947897: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.11\n",
-      "[1,0]<stderr>:2021-04-06 09:25:24.305699: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.11\n",
-      "[1,1]<stderr>:2021-04-06 09:25:24.348176: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.11\n",
-      "[1,0]<stdout>:Step #0\tLoss: 0.692326\n",
-      "[1,0]<stdout>:Step #100\tLoss: 0.658852\n",
-      "[1,0]<stdout>:Step #200\tLoss: 0.667175\n",
-      "[1,0]<stdout>:Step #300\tLoss: 0.664376\n",
-      "[1,0]<stdout>:Step #400\tLoss: 0.662460\n",
-      "[1,0]<stdout>:Step #500\tLoss: 0.665670\n",
-      "[1,0]<stdout>:Step #600\tLoss: 0.651120\n",
-      "[1,0]<stdout>:Step #700\tLoss: 0.653099\n",
-      "[1,0]<stdout>:Step #800\tLoss: 0.648519\n",
-      "[1,0]<stdout>:Step #900\tLoss: 0.645759\n",
-      "^C\n"
+      "[1,0]<stderr>:Traceback (most recent call last):\n",
+      "[1,0]<stderr>:  File \"trainer.py\", line 13, in <module>\n",
+      "[1,0]<stderr>:    import horovod.tensorflow as hvd  # noqa: E402\n",
+      "[1,0]<stderr>:  File \"/conda/envs/merlin/lib/python3.8/site-packages/horovod/tensorflow/__init__.py\", line 24, in <module>\n",
+      "[1,0]<stderr>:    check_extension('horovod.tensorflow', 'HOROVOD_WITH_TENSORFLOW', __file__, 'mpi_lib')\n",
+      "[1,0]<stderr>:  File \"/conda/envs/merlin/lib/python3.8/site-packages/horovod/common/util.py\", line 56, in check_extension\n",
+      "[1,0]<stderr>:    raise ImportError(\n",
+      "[1,0]<stderr>:ImportError: Extension horovod.tensorflow has not been built: /conda/envs/merlin/lib/python3.8/site-packages/horovod/tensorflow/mpi_lib.cpython-38-x86_64-linux-gnu.so not found\n",
+      "[1,0]<stderr>:If this is not expected, reinstall Horovod with HOROVOD_WITH_TENSORFLOW=1 to debug the build error.\n",
+      "[1,1]<stderr>:Traceback (most recent call last):\n",
+      "[1,1]<stderr>:  File \"trainer.py\", line 13, in <module>\n",
+      "[1,1]<stderr>:    import horovod.tensorflow as hvd  # noqa: E402\n",
+      "[1,1]<stderr>:  File \"/conda/envs/merlin/lib/python3.8/site-packages/horovod/tensorflow/__init__.py\", line 24, in <module>\n",
+      "[1,1]<stderr>:    check_extension('horovod.tensorflow', 'HOROVOD_WITH_TENSORFLOW', __file__, 'mpi_lib')\n",
+      "[1,1]<stderr>:  File \"/conda/envs/merlin/lib/python3.8/site-packages/horovod/common/util.py\", line 56, in check_extension\n",
+      "[1,1]<stderr>:    raise ImportError(\n",
+      "[1,1]<stderr>:ImportError: Extension horovod.tensorflow has not been built: /conda/envs/merlin/lib/python3.8/site-packages/horovod/tensorflow/mpi_lib.cpython-38-x86_64-linux-gnu.so not found\n",
```

Comment:
Do we need to re-render this notebook?

Reply:
Yes, see above:
> It will need to be re-run after the error `ModuleNotFoundError: No module named 'nvtabular'` in the TF training container is resolved.

Reply:
makes sense